lib/cloud_controller.rb

```diff
@@ -113,6 +113,7 @@ module VCAP::CloudController; end
 
 require 'cloud_controller/route_binding_message'
 require 'cloud_controller/process_route_handler'
+require 'cloud_controller/copilot_handler'
```

Comment:
naming is hard

some thoughts:
- `process_route_handler` handles process routes
- `copilot_handler` does not handle copilots
- how about `process_route_propogater` in addition to `process_route_handler`
- why not put copilot handler and client together?
- why not combine `copilot_handler` with `process_route_handler`?